src/main/java/step5_racingcar_refactoring/service/Winner.java

```diff
@@ -0,0 +1,54 @@
+package step5_racingcar_refactoring.service;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import step5_racingcar_refactoring.domain.Car;
+import step5_racingcar_refactoring.domain.Cars;
+
+public class Winner {
+
+	Set<Car> winners = new LinkedHashSet<>();
+
+	private int maxValue = 0;
+	public Set<Car> findWinners(Cars cars)
+	{
+		for(Car car : cars.getCars())
+		{
+			setWinner(car);
+		}
+
+		return winners;
+	}
```

Comment:
IDE 의 코드 자동 정렬 기능을 사용해 보면 어떨까요? 😄 
```suggestion
	public Set<Car> findWinners(Cars cars) {
        for (Car car : cars.getCars()) {
            setWinner(car);
        }

        return winners;
    }
```